javaagent-api/src/main/jflex/SqlSanitizer.flex

```diff
@@ -0,0 +1,372 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.api.db;
+
+%%
+
+%final
+%class AutoSqlSanitizer
+%apiprivate
+%int
+%buffer 2048
+
+%unicode
+%ignorecase
+
+COMMA             = ","
+OPEN_PAREN        = "("
+CLOSE_PAREN       = ")"
+OPEN_COMMENT      = "/*"
+CLOSE_COMMENT     = "*/"
+IDENTIFIER        = ([:letter:] | "_") ([:letter:] | [0-9] | [_.])*
```

Comment:
I'm so happy that the abomination I created to represent the unicode letter character class is gone!